protocol/messenger_handler.go

```diff
@@ -1703,7 +1703,7 @@ func (m *Messenger) HandleDeleteForMeMessage(state *ReceivedMessageState, delete
 		return errors.New("chat not found")
 	}
 
-	messagesToDelete, err := m.getConnectedMessages(originalMessage, deleteForMeMessage.LocalChatID)
+	messagesToDelete, err := m.getConnectedMessages(originalMessage, deleteForMeMessage.ChatId)
```

Comment:
Was `LocalChatID` not right? Wouldn't this make it not work correctly in one to one chats?

Reply:
just update assign ChatId to message.LocalChatID, i think the old LocalChatID can be removed? because i can't find where we assigned it? @cammellos